### PR TITLE
ci(workflows): use go 1.19

### PR DESCRIPTION
**Pull request by s-takehana**

### CODE REVIEW

File: generator/types/types.go [hunk 1]
```diff
@@ -37,7 +38,7 @@ func (s String) Tag() string {
 
 // PrismaGoCase transforms `relevance` into `Relevance_`
 func (s String) PrismaGoCase() string {
-	return strings.Title(string(s)) + "_"
+	return cases.Title(language.Und, cases.NoLower).String(string(s)) + "_"
 }
```
Comment (s-takehana) [inline generator/types/types.go:42]: [In Go 1.18 strings.Title() is deprecated. What to use now? And how?](https://stackoverflow.com/questions/71620717/in-go-1-18-strings-title-is-deprecated-what-to-use-now-and-how)

File: runtime/lifecycle/lifecycle.go [hunk 2]
```diff
@@ -13,15 +13,15 @@ type Lifecycle struct {
...
-//   defer func() {
-//     if err := client.Prisma.Disconnect(); err != nil {
-//       panic(fmt.Errorf("could not disconnect: %w", err))
-//     }
-//   }()
+//	defer func() {
+//	  if err := client.Prisma.Disconnect(); err != nil {
+//	    panic(fmt.Errorf("could not disconnect: %w", err))
+//	  }
+//	}()
```
Comment (steebchen) [inline runtime/lifecycle/lifecycle.go:24]: it has to be indented like before, otherwise it'll not render as code in IDEs

File: go.sum [hunk 3]
```diff
@@ -16,6 +16,31 @@ github.com/stretchr/testify v1.8.0 h1:pSgiaMZlXftHpm5L7V1+rVB+AZJydKsMxsQBIJw4PK
...
+golang.org/x/term v0.0.0-20210927222741-03fcf44c2211/go.mod h1:jbD1KX2456YbFQfuXm/mYQcufACuNUgVhRMnK/tPxf8=
+golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
+golang.org/x/text v0.3.3/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
+golang.org/x/text v0.3.7/go.mod h1:u+2+/6zg+i71rQMx5EYifcz6MCKuco9NR6JIITiCfzQ=
+golang.org/x/text v0.4.0 h1:BrVqGRd7+k1DiOgtnFvAkoQEWQvBc25ouMJM6429SFg=
+golang.org/x/text v0.4.0/go.mod h1:mrYo+phRRbMaCq/xk9113O4dZlRixOauAjOtrjsXDZ8=
+golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
+golang.org/x/tools v0.0.0-20191119224855-298f0cb1881e/go.mod h1:b+2E5dAYhXwXZwtnZ6UAqBI28+e2cm9otk0dWdXHAEo=
+golang.org/x/tools v0.1.12/go.mod h1:hNGJHUnrk76NpqgfD5Aqm5Crs+Hm0VOH/i9J2+nxYbc=
+golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7/go.mod h1:I/5z698sn9Ka8TeJc9MKroUUfqBBauWjQqLJ2OPfmY0=
```
Comment (steebchen) [inline go.sum:43]: make sure to run `go mod tidy` if you didn't already

File: generator/types/types.go (hunk 1 above)
Comment (steebchen) [reply generator/types/types.go:42]: good question, It think the solution you chose is fine

**Review by steebchen — APPROVED** [review]
looks good ty just a few changes

File: runtime/lifecycle/lifecycle.go (hunk 2 above)
Comment (s-takehana) [reply runtime/lifecycle/lifecycle.go:24]: I've fixed this [lint error](https://github.com/prisma/prisma-client-go/actions/runs/3275369205/jobs/5390123681#step:8:7).

Please refer to the following.
https://tip.golang.org/doc/go1.19#go-doc

File: go.sum (hunk 3 above)
Comment (s-takehana) [reply go.sum:43]: I've run `go mod tidy`.

Is it related to this?
https://tip.golang.org/doc/go1.18#go-command
> The go mod tidy command now retains additional checksums in the go.sum file for modules whose source code is needed to verify that each imported package is provided by only one module in the build list.

File: runtime/lifecycle/lifecycle.go (hunk 2 above)
Comment (steebchen) [reply runtime/lifecycle/lifecycle.go:24]: Ah thanks, my mistake. Then my editor is just not interpreting it right yet.

File: go.sum (hunk 3 above)
Comment (steebchen) [reply go.sum:43]: all good, just wanted to make sure

**Comment by steebchen** [comment]
Thanks!

**Pull request merged**